Assets/Plugins/Android/Editor/EOSOnPreprocessBuild_android.cs

```diff
@@ -221,4 +223,33 @@ public void ConfigureEOSDependentLibrary()
         }
     }
 
+    private void DetermineLibraryLinkingMethod()
+    {
+        var androidBuildConfigSection = EOSPluginEditorConfigEditor.GetConfigurationSectionEditor<EOSPluginEditorAndroidBuildConfigSection>();
+        androidBuildConfigSection?.Awake();
+        if (androidBuildConfigSection != null)
+        {
+            if (androidBuildConfigSection.GetCurrentConfig() == null)
+            {
+                androidBuildConfigSection.LoadConfigFromDisk();
+            }
+        }
+
+        string packagePathname = Path.GetFullPath("Packages/" + GetPackageName() + "/PlatformSpecificAssets~/EOS/Android/");
+        string libSubpath = "/aar/eos-sdk.aar";
+
+        string dynamicPathName = Path.Combine(Application.dataPath, packagePathname, "dynamic-stdc++" + libSubpath);
+        string staticPathName = Path.Combine(Application.dataPath, packagePathname, "static-stdc++" + libSubpath);
```

Comment:
This is checking for the paths incorrectly.
It needs check if the libraries exist in the `packagePathname` directory, and if they don't, assume they exist in the `../PlatformSpecificAssets/EOS/Android` path and copy them into `destPathname`.